config/routes.rb

```diff
@@ -1,10 +1,24 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
-  root 'application#index'
+  resources :attractions, only: [:index]
+
+  resources :users
+
+  resources :home
+
+  get '/users/:id/categories' => 'users#travel_profile', as: :travel_profile
+
+  get "/log-in" => "sessions#new"
+  post "/log-in" => "sessions#create"
+  get "/log-out" => "sessions#destroy", as: :log_out
+
+  get 'maps/start' => 'maps#start'
+  get 'maps/directions' => 'maps#directions'
+  post 'maps/search' => 'maps#search'
+  resources :maps, only: [:index]
```

Comment:
Again, you can move start, directions, and search into the resources: :exclamation: 
```ruby
resources :maps, only: [:index] do
  collection do
    get 'start'
    get 'directions'
    get 'search'
  end
end
```